Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataEntity.cs

```diff
@@ -155,6 +155,6 @@ public IEnumerable<string> Rosters
         /// <summary>
         /// Gets or sets the sending started DateTime value.
         /// </summary>
-        public DateTime? SendingStartedDate { get; set; }
+        public DateTime? SendingStartedDateTime { get; set; }
```

Comment:
Remove this change - although it is a DateTime, we are going to not require the type to be in the name for the entity (SentDate, etc.). Also, we need to be wary of changes like this because, for users that already have the app deployed, this could be a breaking change.

Reply:
Addressed.